src/lib/Components/Bidding/Screens/BillingAddress.tsx

```diff
@@ -232,4 +280,21 @@ export class BillingAddress extends React.Component<BillingAddressProps, Billing
       value: this.state.values[field],
     }
   }
+
+  private yPosition({ y, height }) {
+    const windowHeight = Dimensions.get("window").height
+
+    return Math.max(0, y - windowHeight + height + iOSAccessoryViewHeight + this.keyboardHeight + this.iPhoneXOffset)
+  }
+
+  private get verticalOffset() {
+    return this.iPhoneXOffset + 15
+  }
+
+  // TODO: Remove this once React Native has been updated
```

Comment:
I like that this comment is here 👍 I think it could be more specific – if there's an issue we can link to or a version number we can specify, I think that'd be a good addition here.

Reply:
@orta Do you know any links we can specify here?

Reply:
It is iPhone X support stuff, not working around a specific bug / implementation issue - so updating RN will fix it